tests/test.js

```diff
@@ -56,52 +56,37 @@ describe("Embroider build", function () {
 });
 
 describe("When rootURL and publicPath do not match", function () {
-  let indexHtml, manifest, favicons, stylesheets;
+  let indexHtml, stylesheets;
 
   before(async function () {
     indexHtml = await getIndexHtml("rooturl-and-publicpath-differ");
-    favicons = indexHtml.querySelectorAll(
-      'link[rel="icon"], link[rel="apple-touch-icon"]',
-    );
-    manifest = indexHtml.querySelector('link[rel="manifest"]');
     stylesheets = indexHtml.querySelectorAll('link[rel="stylesheet"]');
   });
 
-  it("Favicons have integrity hash", function () {
-    for (const favicon of favicons) {
-      expect(favicon.hasAttribute("integrity")).to.be.true;
-      expect(favicon.getAttribute("integrity")).to.be.a("string");
-    }
-  });
-
-  it("Favicons have crossorigin attribute", function () {
-    for (const favicon of favicons) {
-      expect(favicon.hasAttribute("crossorigin")).to.be.true;
-      expect(favicon.getAttribute("crossorigin")).to.be.a("string");
-      expect(favicon.getAttribute("crossorigin")).to.equal("anonymous");
-    }
-  });
-
-  it("Manifest has integrity hash", function () {
-    expect(manifest.hasAttribute("integrity")).to.be.true;
-    expect(manifest.getAttribute("integrity")).to.be.a("string");
-  });
-
-  it("Manifest has crossorigin attribute", function () {
-    expect(manifest.hasAttribute("crossorigin")).to.be.true;
-    expect(manifest.getAttribute("crossorigin")).to.be.a("string");
-    expect(manifest.getAttribute("crossorigin")).to.equal("anonymous");
-  });
-
   it("Some links start with publicPath, some start with rootURL", function () {
     for (const stylesheet of stylesheets) {
       const href = stylesheet.getAttribute("href");
       expect(href.startsWith("/a-different-public-path/")).to.be.true;
     }
+  });
+});
 
-    for (const favicon of favicons) {
-      const href = favicon.getAttribute("href");
-      expect(href.startsWith("/a-root-url/")).to.be.true;
+describe("Link tags with valid rel attributes are processed", function () {
+  let indexHtml, links;
+  before(async function () {
+    indexHtml = await getIndexHtml("link-rel-without-integrity-hash-support");
+    links = indexHtml.querySelectorAll("link");
+  });
+  it(`Processes link when rel is 'module', 'modulepreload', or 'stylesheet'`, function () {
+    for (const link of links) {
+      const rel = link.getAttribute("rel");
+      const integrity = link.getAttribute("integrity");
+      if (rel === "module" || rel === "stylesheet" || rel === "modulepreload") {
```

Comment:
Oops, realized I should add in an alternate stylesheet and swap this to string.includes